packages/core/src/utils.ts

```diff
@@ -0,0 +1,3 @@
+export function isObjectLiteral(item: unknown): item is { [key: string]: unknown } {
```

Comment:
Nit - better naming

```suggestion
export function isObjectLiteral(candidate: unknown): candidate is { [key: string]: unknown } {
```

Reply:
we have an exact copy of this in `cli`. if we rename this, we should rename it in the other copy as well, so that if/when we deduplicate this code, it's not confusing for the person doing the refactoring.

Reply:
Since we're here let's keep only one copy in `core` and import it from elsewhere?